CONTRIBUTING.md

```diff
@@ -40,7 +40,7 @@ If you need to sync commits from `robocomp/robocomp` to push your own changes, f
 be sure to update the `Readme.md` installation tutorial to include your changes.
 
 - If you plan to make big changes, ensure you are not breaking Robocomp's core functionalities.
-Even through Travis tests the build, you could break an internal function. 
+Even though Travis tests the build, you could break an internal function. 
 Please ensure you test new features locally at first.
```

Comment:
```suggestion
Please ensure you test new features locally at first and add some unit tests for the new functionalities.
```

Reply:
Thank you for your contribution and for pointing to this piece of the documentation.

Reply:
Welcome, @orensbruli.